lib/oli_web/components/delivery/page_delivery.ex

```diff
@@ -18,15 +20,22 @@ defmodule OliWeb.Components.Delivery.PageDelivery do
     """
   end
 
+  attr :read_by, Date, default: nil
+  attr :est_reading_time, Timex.Duration, default: nil
+
   def details(assigns) do
     ~H"""
       <div class="flex flex-row my-2">
-        <div class="py-1.5 px-4 bg-gray-100 text-gray-700 rounded">
-          Read by 10-03-2022
-        </div>
-        <div class="py-1.5 px-4 bg-gray-100 text-gray-700 rounded ml-1">
-          Estimated reading time: 5 mins
-        </div>
+        <%= if @read_by do %>
+          <div class="py-1.5 px-4 bg-gray-100 text-gray-700 rounded">
+            Read by <%= format_date(@read_by) %>
```

Comment:
The soft scheduled `end_date`, which is what I believe is populating the `read_by` field here, isn't always "Read By".  On the `SectionResource` record, there is a `scheduling_type` attribute which can be either `:read_by` or `:inclass_activity`, which should be labeled "Ready by" or "In class activity"

Reply:
Addressed, added case check for scheduling type and connected up the section's soft schedule